docs/next.config.js

```diff
@@ -136,22 +136,22 @@ module.exports = withNextPluginPreval({
       },
       /**
        * source: a url has one of the folder's names (components, getting-started, guides, theming)
-       * destination: add '[platform]' to the the beginning
+       * destination: add '/react/' to the the beginning
        */
       {
         source:
           '/:nav(legacy-components|connected-components|components|getting-started|guides|theming)/:page*',
-        destination: '/[platform]/:nav/:page*',
+        destination: '/react/:nav/:page*',
```

Comment:
Doesn't this break the other platforms?

Reply:
No it won't. Before it was 

"no platform" -> "[platform]" -> "default platform (react)", 

and now it's

"no platform" -> "react (default platform)"